omnibus/lib/ostools.rb

```diff
@@ -67,3 +67,7 @@ def with_python_runtime?(runtime)
     python_runtimes = ENV['PY_RUNTIMES'].nil? ? ['3'] : ENV['PY_RUNTIMES'].split(',')
     return python_runtimes.include? runtime
 end
+
+def fips_mode?()
+    return ENV['FIPS_MODE'] == 'true' && (linux_target? || windows_target?)
```

Comment:
Could you add this new environment variable to `excluded_values` in `./tasks/libs/common/omnibus.py`? This doesn't impact the native builds and we can share the omnibus cache regardless

Reply:
ae92f6449768a64b8417307605f499406000fda5